File: app/src/main/java/nithra/tamil/word/game/giftsuggestions/Retrofit/AddGift.java
package nithra.tamil.word.game.giftsuggestions.Retrofit;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AddGift {
    @SerializedName("id")
    @Expose
    public String id;
    @SerializedName("user_id")
    @Expose
    public String userId;
    @SerializedName("gift_category")
    @Expose
    public String giftCategory;
    @SerializedName("gift_for")
    @Expose
    public String giftFor;
    @SerializedName("gift_name")
    @Expose
    public String giftName;
    @SerializedName("gift_image")
    @Expose
    public String giftImage;
    @SerializedName("gift_amount")
    @Expose
    public String giftAmount;
    @SerializedName("discount")
    @Expose
    public String discount;
    @SerializedName("total_amount")
    @Expose
    public String totalAmount;
    @SerializedName("gift_description")
    @Expose
    public String giftDescription;
    @SerializedName("gift_status")
    @Expose
    public String giftStatus;
    @SerializedName("status")
    @Expose
    public String status;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getGiftCategory() {
        return giftCategory;
    }

    public void setGiftCategory(String giftCategory) {
        this.giftCategory = giftCategory;
    }

    public String getGiftFor() {
        return giftFor;
    }

    public void setGiftFor(String giftFor) {
        this.giftFor = giftFor;
    }

    public String getGiftName() {
        return giftName;
    }

    public void setGiftName(String giftName) {
        this.giftName = giftName;
    }

    public String getGiftImage() {
        return giftImage;
    }

    public void setGiftImage(String giftImage) {
        this.giftImage = giftImage;
    }

    public String getGiftAmount() {
        return giftAmount;
    }

    public void setGiftAmount(String giftAmount) {
        this.giftAmount = giftAmount;
    }

    public String getDiscount() {
        return discount;
    }

    public void setDiscount(String discount) {
        this.discount = discount;
    }

    public String getTotalAmount() {
        return totalAmount;
    }

    public void setTotalAmount(String totalAmount) {
        this.totalAmount = totalAmount;
    }

    public String getGiftDescription() {
        return giftDescription;
    }

    public void setGiftDescription(String giftDescription) {
        this.giftDescription = giftDescription;
    }

    public String getGiftStatus() {
        return giftStatus;
    }

    public void setGiftStatus(String giftStatus) {
        this.giftStatus = giftStatus;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: app/src/main/java/nithra/gift/suggestion/shop/birthday/marriage/retrofit/AddGift.kt
package nithra.gift.suggestion.shop.birthday.marriage.retrofit

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class AddGift {
    @SerializedName("id")
    @Expose
    var id: String? = null

    @SerializedName("user_id")
    @Expose
    var userId: String? = null

    @SerializedName("gift_category")
    @Expose
    var giftCategory: String? = null

    @SerializedName("gift_for")
    @Expose
    var giftFor: String? = null

    @SerializedName("gift_name")
    @Expose
    var giftName: String? = null

    @SerializedName("gift_image")
    @Expose
    var giftImage: String? = null

    @SerializedName("gift_amount")
    @Expose
    var giftAmount: String? = null

    @SerializedName("discount")
    @Expose
    var discount: String? = null

    @SerializedName("total_amount")
    @Expose
    var totalAmount: String? = null

    @SerializedName("gift_description")
    @Expose
    var giftDescription: String? = null

    @SerializedName("gift_status")
    @Expose
    var giftStatus: String? = null

    @SerializedName("status")
    @Expose
    var status: String? = null
}